### ISSUE TITLE
The error occurs when loading the training data with the DataLoader.

### ISSUE THREAD

**Issue by pikanzz** [KwaiVGI/VideoAlign#5]
Thank you for your excellent open-source project, and it has helped me solve many problems.
When I load the training data from the repo, the training works fine, but when I use the validation set as the training set, I encounter the following error.

![Image](https://github.com/user-attachments/assets/4ea824a6-09c6-45b5-865f-b323dc35c782)

![Image](https://github.com/user-attachments/assets/7cebdd1d-2580-4d35-af4e-44b45b10b7d8)
It seems that it’s not an issue with SHM.

**Comment by GongyeLiu** [comment]
Hi @pikanzz ,
I suspect the error may be related to the num_workers in torch's Dataloader. I haven't encountered a similar issue before, but it might be still related to the SHM allocation in Docker, I suspect. You could try setting num_workers to a lower value, or even to 0, to determine if the issue is related to multiprocessing. 
I'm not sure if my suggestion is correct, but I hope it helps.

**Comment by pikanzz** [comment]
Thank you for your early reply！
I have tried multiple num_worker settings, including 0, 1, 2, and 8, but still encountered the above issue. Additionally, I have checked whether shm is insufficient and confirmed that shm is sufficient.

![Image](https://github.com/user-attachments/assets/3221c475-3545-4d18-bbd8-8ffc757de7eb)
I tried using https://huggingface.co/datasets/KwaiVGI/VideoGen-RewardBench
 as the training set and modified its format to the required format. So far, I have tested extracting the first 20 rows of VideoGen-RewardBench, which can be trained normally. However, when loading the entire CSV file, an error occurs. I used 8*h20.

**Comment by pikanzz** [comment]
![Image](https://github.com/user-attachments/assets/c6047527-e366-4295-91e5-063b970264b0)
Hi @yifan123 ,error as pic shows

**Comment by pikanzz** [comment]
@GongyeLiu @yifan123 How can I train only “Motion Quality” and freeze the other parts of the model?
